packages/pebble-shared/icons.json

```diff
@@ -206,6 +209,7 @@
   "site-visit-2",
   "site-visit",
   "sms",
+  "source",
```

Comment:
dont use [source](https://materialui.co/icon/source) for icon name because we use the word "source" to denote lead source. this creates implicit connection with source and subsource - should there be a "subsource" icon? what should it look like?

Reply:
we could use [topic](https://materialui.co/icon/topic) to match material icons exactly but it allows adding [folder](https://materialui.co/icon/folder) icon in future which looks about the same.

our goal is to keep icon names generic and icon designs largely unique to avoid situation where similar designs have very different names (like "phone" and "call") and when similar names have very different designs (like "arrow")

check these icons in pebble website:
https://pebble-anarock.netlify.app/?path=/story/theme-icons--list

Reply:
This was the name in material ui file theme

Reply:
can we name this icon to folder-dashed or something like that ?

Reply:
any reason to be specific about type of folder?

Reply:
No i thought folder icon name already exists in pebble

Reply:
u can check pebble icons in repo or on [website](https://pebble-anarock.netlify.app/?path=/story/theme-icons--list)